apps/wallet/src/engine/runners/solana/metadata.ts

```diff
@@ -59,9 +38,8 @@ const getRemoteMeta: GetMetadataFunc = async (connection, mintAddress) => {
 	if (!info?.data) return result;
 
 	const [metadata] = Metadata.deserialize(info.data);
-	result.name = metadata.data?.name;
-	result.symbol = metadata.data?.symbol;
-	result.mpl = metadata;
+	result.name = metadata.data?.name.replaceAll('\u0000', '');
+	result.symbol = metadata.data?.symbol.replaceAll('\u0000', '');
```

Comment:
this kind of logic should be inside util/shared method, e.g `utils.escapeFixedBufferString(metadata?.data?.name)`